includes/Widget/Menu.php

```diff
@@ -117,7 +117,14 @@ public function widget( $args, $instance )  {
 
 			$menu_ul_id = str_replace( '_', '-', $instance['bgc_menu_location_id'] ) . '-menu';
 
-			echo '<div id="' . $instance['bgc_menu_location_id'] . '-wrap" class="bgtfw-menu-wrap">';
+			$ham_control_id = 'bgtfw_menu_hamburger_display_' . str_replace( '_', '-', $instance['bgc_menu_location_id'] );
+			$ham_control_id = preg_replace( '/-(\d{3})/', '_$1', $ham_control_id );
+
+			$ham_class = implode( ' ', get_theme_mod( $ham_control_id, array( 'ham-phone' ) ) );
+			$class    .= ' ' . $ham_class;
+
+
+			echo '<div id="' . $instance['bgc_menu_location_id'] . '-wrap" class="bgtfw-menu-wrap ' . $ham_class . ' ' . $align . '">';
 
 			// Make sure that if there is a registerd location already, that it is used.
 			$menu_id = isset( $registered_locations[ $instance['bgc_menu_location_id'] ] )
```

Comment:
Does this need to have a semi colon at the end of the line?

Reply:
No.  It's part of a ternary operation; see the next two lines.